### DISCUSSION TITLE
Backup and restore to NFS Server

### DISCUSSION THREAD

**Discussion by FaraSys** [longhorn/longhorn#5181] [Q&A] [answered]
Hi everyone.
I tested backup my pv/pvc using NFS server. 
I stop my pod and delete pv via GUI Longhorn. Next I choose my backup file and click **Restore Latest Backup.**
After restore, I choose my Volume and click Create PV. Everythings ok. My pv/pvc work fine after start my pod.
![image](https://user-images.githubusercontent.com/67997008/210058224-67a66990-2d89-4f91-a7f9-fc174c908307.png)
But for pv StorageClass has changed. Why? I have one storageClass and no longhorn-static. Is this normal behavior? see screen
![image](https://user-images.githubusercontent.com/67997008/210058488-b6b9c0b0-2e44-40a2-bdb7-2e82cbf33268.png)

**Comment by mantissahz** [comment, marked as answer]
Hi @FaraSys, It's a normal behavior.
Longhorn uses the default Longhorn Static StorageClass Name longhorn-static when PV/PVC is created.

**Reply by FaraSys** [reply]
Thanks for clarifying.
